frontend/ui/lib/main.dart

```diff
@@ -34,6 +37,18 @@ class TripMeOut extends StatelessWidget {
             settings: settings,
           );
         }
+        if (settings.name == '/createTrip') {
```

Comment:
Consider adding links to these new routes from the current index page (`/trips`) to make it possible to see these new pages by clicking around. As of now, you need to enter the route names in the URL bar to do this.

Note: don't bother adding these navigation links if you don't think `/trips` will continue to act as the index page, though in that case you might want to add a new default route.

Reply:
Dont do this in this cl.

IMO /createTrip should be linked from /trips when you add new (follows our initial skeleton diagram)

Reply:
That makes sense. My understanding was that there would be a single view-trip page; using the nested structure you suggest for routing (matching the REST endpoints) changes that.